packages/actionmenu/src/react/index.tsx

```diff
@@ -66,16 +66,8 @@ export const ActionMenu = React.forwardRef<HTMLUListElement, ActionMenuProps>(
       children,
       ...rest
     },
-    forwardedRef
+    ref
   ) => {
-    const ref = React.useRef<HTMLUListElement | null>(null)
```

Comment:
By removing this, we are removing the functionality to close the actionmenu by clicking outside of it.  That doesn't seem like a UX improvement. At the least it seems like it's a breaking change.

Why remove this entirely instead of changing away from the buggy implementation?

Reply:
Considered that actually but still couldn't get it to work because of Position not the hook. I could try and fix Position but that might cause unforeseen breakage. It's basically too coupled.

Reply:
I don't understand this answer.  Perhaps I need more info in order to:

How are Position and useCloseOnDocumentEvents too coupled?

And I don't understand the comment in standup:

Why does Position or ActionMenu need broken?  And what does broken mean?